contrib/navidrome.service

```diff
@@ -11,15 +11,13 @@ WantedBy=multi-user.target
 User=navidrome
 Group=navidrome
 Type=simple
-ExecStart=/usr/bin/navidrome
+ExecStart=/usr/bin/navidrome --configfile "/etc/navidrome/navidrome.toml"
```

Comment:
Should be `navidrome service execute --configfile ...` now

Reply:
The idea is to not use this scripts, but use `navidrome service install` to auto-generate them.

Also, the syntax above should still work for Linux and macOS, right?

Reply:
Yeah it should, but I think it makes sense for us to prefer to run inside the Kardianos wrapper so we have consistent behaviour between OS's?